### ISSUE TITLE
Salesforce plugin : the test on www.howsmyssl.com should fail silently

### ISSUE THREAD

**Issue by jereze** [dataiku/dataiku-contrib#76]
```
09:30:07] [INFO] [dku.flow.activity] - Run thread for activity compute_salesforce_token_NP starting
[09:30:07] [INFO] [dku.flow.custompython] - Dumping Python script to /data.ssd/analytics/dssprod/jobs/SALESFORCE_RAWDATA/sched_build_2018-06-20T07-30-06.959/compute_salesforce_token_NP/custom-python-recipe/pyoutSpvoRXKAXFtx/script.py
[09:30:08] [INFO] [dip.venv.selector] - Select in plugin with {"config":{}}
[09:30:08] [INFO] [dku.code.envs.resolution] - Executing Python activity in builtin env
[09:30:08] [INFO] [dku.datasets.file] - Building Filesystem handler config: {"connection":"filesystem_managed","path":"/SALESFORCE_RAWDATA.salesforce_token","notReadyIfEmpty":false,"filesSelectionRules":{"mode":"ALL","excludeRules":[],"includeRules":[],"explicitFiles":[]}}
[09:30:08] [INFO] [dku.security.process] - Starting process (regular)
[09:30:08] [INFO] [dku.security.process] - Process started with pid=19207
[09:30:08] [INFO] [dku.recipes.code.base] - Process reads from nothing
[09:30:13] [INFO] [dku.utils]  - Debug TLS:
[09:30:13] [INFO] [dku.utils]  - *************** Recipe code failed **************
[09:30:13] [INFO] [dku.utils]  - Begin Python stack
[09:30:13] [INFO] [dku.utils]  - Traceback (most recent call last):
[09:30:13] [INFO] [dku.utils]  -   File "/data.ssd/analytics/dssprod/jobs/SALESFORCE_RAWDATA/sched_build_2018-06-20T07-30-06.959/compute_salesforce_token_NP/custom-python-recipe/pyoutSpvoRXKAXFtx/python-exec-wrapper.py", line 6, in <module>
[09:30:13] [INFO] [dku.utils]  -     exec(f.read())
[09:30:13] [INFO] [dku.utils]  -   File "<string>", line 46, in <module>
[09:30:13] [INFO] [dku.utils]  -   File "/data.ssd/analytics/dssprod/pyenv/lib/python2.7/site-packages/requests/api.py", line 72, in get
[09:30:13] [INFO] [dku.utils]  -     return request('get', url, params=params, **kwargs)
[09:30:13] [INFO] [dku.utils]  -   File "/data.ssd/analytics/dssprod/pyenv/lib/python2.7/site-packages/requests/api.py", line 58, in request
[09:30:13] [INFO] [dku.utils]  -     return session.request(method=method, url=url, **kwargs)
[09:30:13] [INFO] [dku.utils]  -   File "/data.ssd/analytics/dssprod/pyenv/lib/python2.7/site-packages/requests/sessions.py", line 508, in request
[09:30:13] [INFO] [dku.utils]  -     resp = self.send(prep, **send_kwargs)
[09:30:13] [INFO] [dku.utils]  -   File "/data.ssd/analytics/dssprod/pyenv/lib/python2.7/site-packages/requests/sessions.py", line 618, in send
[09:30:13] [INFO] [dku.utils]  -     r = adapter.send(request, **kwargs)
[09:30:13] [INFO] [dku.utils]  -   File "/data.ssd/analytics/dssprod/pyenv/lib/python2.7/site-packages/requests/adapters.py", line 506, in send
[09:30:13] [INFO] [dku.utils]  -     raise SSLError(e, request=request)
[09:30:13] [INFO] [dku.utils]  - SSLError: HTTPSConnectionPool(host='www.howsmyssl.com', port=443): Max retries exceeded with url: /a/check (Caused by SSLError(CertificateError("hostname 'www.howsmyssl.com' doesn't match either of 'darkishgreen.com', 'jenkins.internal.darkishgreen.com', 'somethingsimilar.com', 'staging-subscriptions.howsmyssl.com', 'staging.darkishgreen.com', 'staging.somethingsimilar.com', 'stripetest-subscriptions.howsmyssl.com', 'subscriptions.howsmyssl.com', 'whoownsmyavailability.com', 'whoownsyouravailability.com', 'www.darkishgreen.com', 'www.somethingsimilar.com', 'www.whoownsmyavailability.com', 'www.whoownsyouravailability.com'",),))
[09:30:13] [INFO] [dku.utils]  - End Python stack
```

**Comment by maludwig** [comment]
I agree. It's not ideal that I must open up the HTTPS from the firewall to www.howsmyssl.com, it feels weird, and while it provides nice debug data, it shouldn't be a source of bugs itself.

**Comment by jereze** [comment]
Fixed in 1.2.0. https://github.com/dataiku/dss-plugin-salesforce
